src/main/java/org/jitsi/jicofo/FocusManager.java

```diff
@@ -360,14 +363,18 @@ private void createConference(
             = JitsiMeetGlobalConfig.getGlobalConfig(
                 FocusBundleActivator.bundleContext);
 
-        String id = generateConferenceId();
-        JitsiMeetConferenceImpl conference
-            = new JitsiMeetConferenceImpl(
+        JitsiMeetConferenceImpl conference;
+        String id;
+        synchronized (conferencesSyncRoot)
+        {
+            id = generateConferenceId();
```

Comment:
what i meant was that `generateConferenceId` already grabs the lock inside, so calling `id = generateConferenceId()` outside this synchronized block will still generate the conference id safely.